presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/PackageAPI.java

```diff
@@ -141,7 +147,7 @@ public void run()
                 command.runCommand();
             }
             catch (Exception e) {
-                // TODO: Add to logger
+                LOGGER.error("Encountered exception", e);
```

Comment:
This one catch all the PrestoManagerException? If so, you should give more informative message, i.e. record the e.message in the log